native/src/components/Header.tsx

```diff
@@ -83,6 +85,7 @@ const Header = ({
   // Save route/canGoBack to state to prevent it from changing during navigating which would lead to flickering of the title and back button
   const [previousRoute] = useState(navigation.getState().routes[navigation.getState().routes.length - 2])
   const [canGoBack] = useState(navigation.canGoBack())
+  const { enabled: isTtsActive, setVisible: setTtsPlayerVisible } = useTtsPlayer()
```

Comment:
:upside_down_face: Please keep the naming consistent
```suggestion
  const { enabled: isTtsEnabled, setVisible: setTtsPlayerVisible } = useTtsPlayer()
```

Reply:
This is still valid